dask_ml/model_selection/_hyperband.py

```diff
@@ -0,0 +1,557 @@
+        this value, and if changed we recommend ``eta=4``.
+        Some theory behind Hyperband suggests ``eta=np.e``. Higher
+        values imply higher confidence in model selection.
+
+    test_size : float
+        Fraction of the dataset to hold out for computing test scores.
+        Defaults to the size of a single partition of the input training set
+
+        .. note::
+
+           The testing dataset should fit in memory on a single machine.
+           Adjust the ``test_size`` parameter as necessary to achieve this.
+
+    random_state : int, RandomState instance or None, optional, default: None
+        If int, random_state is the seed used by the random number generator;
+        If RandomState instance, random_state is the random number generator;
+        If None, the random number generator is the RandomState instance used
+        by `np.random`.
+
+    scoring : string, callable, list/tuple, dict or None, default: None
+        A single string (see :ref:`scoring_parameter`) or a callable
+        (see :ref:`scoring`) to evaluate the predictions on the test set.
+
+        For evaluating multiple metrics, either give a list of (unique) strings
+        or a dict with names as keys and callables as values.
+
+        NOTE that when using custom scorers, each scorer should return a single
+        value. Metric functions returning a list/array of values can be wrapped
+        into multiple scorers that return one value each.
+
+        See :ref:`multimetric_grid_search` for an example.
+
+        If None, the estimator's default scorer (if available) is used.
+
+    patience : int, default False
+        Maximum number of non-improving scores before we stop training a
+        model. Off by default.
+
+    tol : float, default 0.001
+        The required level of improvement to consider stopping training on
+        that model. The most recent score must be at at most ``tol`` better
+        than the all of the previous ``patience`` scores for that model.
+        Increasing ``tol`` will tend to reduce training time, at the cost
+        of worse estimators.
+
+    **kwargs : dict, optional
+        Parameters to pass to
+        :func:`~dask_ml.model_selection.IncrementalSearchCV`
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from dask_ml.model_selection import HyperbandSearchCV
+    >>> from dask_ml.datasets import make_classification
+    >>> from sklearn.linear_model import SGDClassifier
+    >>>
+    >>> X, y = make_classification(chunks=20)
+    >>> est = SGDClassifier(tol=1e-3)
+    >>> param_dist = {'alpha': np.logspace(-4, 0, num=1000),
+    >>>               'loss': ['hinge', 'log', 'modified_huber', 'squared_hinge'],
+    >>>               'average': [True, False]}
+    >>>
+    >>> search = HyperbandSearchCV(est, param_dist)
+    >>> search.fit(X, y, classes=np.unique(y))
+    >>> search.best_params_
+    {'loss': 'log', 'average': False, 'alpha': 0.0080502}
+
+    """
+    + INC_ATTRS
```

Comment:
Let's just duplicate this text for now. There are more robust ways to share docstrings, but that's out of scope for now.